modules/tinymce/src/core/main/ts/fmt/FormatTypes.ts

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
Just checking - the only change was adding `inline` to `SelectorFormat`, right? The rest was just moved?

Reply:
Right, I forgot that wouldn't show that change sorry. Yes, adding `inline?: boolean` was the only change.

Reply:
See: [discussion below](https://github.com/tinymce/tinymce/pull/6085/files#r501590850) inline is no longer in SelectorFormat, so I guess it's all just moved?

Reply:
Yeah, that's correct. The only change is that it's moved now :+1: